PrReviewCode/Program.cs

```diff
@@ -4,6 +4,8 @@ public class Program
 {
     public static void Main(string[] args)
     {
+        throw new Exception("Test Exception");
```

Comment:
Uncaught exception in `Main` terminates application startup. This violates fault tolerance and exposes internal stack traces.

 **SUGGESTION:**

```suggestion
 // Handle or remove this exception
```